Solutions/NetClean ProActive/Analytic Rules/NetClean_Sentinel_analytic_rule.yaml

```diff
@@ -0,0 +1,56 @@
+id: "[concat(resourceId('Microsoft.OperationalInsights/workspaces/providers', parameters('workspace'),
+    'Microsoft.SecurityInsights'),'/alertRules/77aaeec4-c852-4d4e-8a93-5a8c35159557')]"
+  name: "[concat(parameters('workspace'),'/Microsoft.SecurityInsights/77aaeec4-c852-4d4e-8a93-5a8c35159557')]"
+  type: Microsoft.OperationalInsights/workspaces/providers/alertRules
+  kind: Scheduled
+  apiVersion: 2022-09-01-preview
```

Comment:
@NCsteven, apiversion is not needed to be added please just add version. Thanks.